TrustModel.md

```diff
@@ -0,0 +1,117 @@
+identifying where there may be considerations to preserve the value of using a TEE.
+
+<!--
+   Follow on pull requests will list here subsequent sections corresponding to orthogonal ways 
+of considering cloud native approaches
+Specifically including
+   - Personas
+   - Lifecycle
+   - Trusted Execution Boundary Location
+   - Secured v unsecured TEE Base Image
+-->
+
+
+Further documentation will highlight specific threat vectors in detail, considering risk, 
+impact, mitigation etc as the project progresses. The Security Assurance section, Page 31, of 
+CNCF Cloud Native Security Whitepaper[^2] will guide this more detailed threat vector effort.
+
+
+### Related Prior Effort
+
+Confidential Containers brings confidential computing into a cloud native context and should 
+therefore refer to and build on trust and security models already defined.
+
+For example: 
+
+- Confidential Computing Consortium (CCC) published 
+  "[A Technical Analysis of Confidential Computing](https://confidentialcomputing.io/wp-content/uploads/sites/85/2021/03/CCC-Tech-Analysis-Confidential-Computing-V1.pdf) [^1]" 
+  section 5 of which defines the threat model for confidential computing.
+- Cloud Native Computing Foundation Security Technical Advisory Group published 
+  "[Cloud Native Security Whitepaper](https://github.com/cncf/tag-security/blob/main/security-whitepaper/CNCF_cloud-native-security-whitepaper-Nov2020.pdf) [^2]" 
+- Kubernetes provides documentation on
+  [an overview of cloud native security](https://kubernetes.io/docs/concepts/security/overview/) [^3]
+- Open Web Application Security Project -
+  [Docker Security Threat Modeling](https://github.com/OWASP/Docker-Security/blob/main/001%20-%20Threats.md) [^4]
+
+  
+The commonality between confidential containers project and confidential computing is to reduce
+the ability for unauthorised access to data and code inside TEEs sufficiently such that this path 
+is not an economically or logically viable attack during
+execution (5.1 Goal)[^1].
+
+This means our trust and threat modelling should 
+- Focus on confidentiality and integrity of data and code.
+- Focus on enhancing existing Cloud Native models in the context of exploiting Trusted 
+  Execution Environments.
+- Consider existing Cloud Native technologies and the role they can play for confidential containers.
+- Consider additional technologies to fulfil a role in Cloud Native exploitation of TEE's.
+
+### Out of Scope
+
+The following items are considered out-of-scope for the trust/threat modelling within confidential 
+containers : 
+
+- Vulnerabilities within the application/code which has been requested to run inside a Trusted 
+  Execution Environment. 
+- Availability part of the Confidentiality/Integrity/Availability in CIA Triad.
+- Software TEEs. At this time we are focused on hardware TEEs.
+- Certain security guarantees are defined by the underlying Trusted Execution Environment and these 
```

Comment:
Nit:  Using the expanded form `Trusted Execution Environment (TEE)` for the first occurrence and consistently using TEE/TEEs for all other occurrences?

Reply:
Thanks, fixed